http/swagger.yml

```diff
@@ -8445,6 +8600,94 @@ components:
     TaskStatusType:
       type: string
       enum: [active, inactive]
+    Invite:
+      properties:
+        id:
+          description: the idpe id of the invite
+          readOnly: true
+          type: string
+        email:
+          type: string
+        role:
+          type: string
+          enum:
+            - member
+            - owner
+        withdrawnAt:
+          format: date-time
+          type: string
+        acceptedAt:
+          format: date-time
+          type: string
```

Comment:
We don't want to expose `withdrawnAt` and `acceptedAt` here; they're internal implementation details of the DB schema and shouldn't be part of the API.